app/models/age_range_option.rb

```diff
@@ -0,0 +1,2 @@
+class AgeRangeOption < ApplicationRecord
```

Comment:
If there's not even enough to this model to write any model methods, maybe it can just be a value in the user table that corresponds to an enum in the model?